src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -1124,50 +1129,75 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
       if (!!resultListContributor) {
         const idFieldName = this.collectionToDescription.get(resultListContributor.collection).id_path;
         const id = feature.properties[idFieldName.replace(/\./g, '_')];
-        resultListContributor.detailedDataRetriever.getData(id).subscribe(
-          data => {
-            const rowItem = new Item([], new Map());
-            rowItem.identifier = id;
-            resultListContributor.detailedDataRetriever.getActions(rowItem).subscribe(ac => rowItem.actions = ac);
-            const detail = new Array<{
-              group: string;
-              details: Array<{
-                key: string;
-                value: string;
-              }>;
-            }>();
-            data.details.forEach((k, v) => {
-              const details = new Array<{
-                key: string;
-                value: string;
-              }>();
-              k.forEach((value, key) => {
-                details.push(
-                  { key, value }
-                );
-              });
-              detail.push({
-                group: v,
-                details: details
-              });
-            });
-            rowItem.itemDetailedData = detail;
-            const popupContent = this.dynamicComponentService.injectComponent(
-              ResultDetailedItemComponent,
-              x => {
-                x.rowItem = rowItem;
-                x.actionOnItemEvent = this.actionOnPopup;
-                x.idFieldName = idFieldName;
-              });
-            if (!!this.popup) {
-              this.popup.remove();
+        // Open the list panel if it's close
+        this.disableRecalculateExtend = true;
+        if (!this.resultlistService.listOpen) {
+          this.toggleList();
+        }
+        // Selec the good tab if we have several one
+        // No tabs case
+        if (this.resultlistContributors.length === 1) {
+          setTimeout(() => {
```

Comment:
why do you need a timeout ?

Reply:
Is there a way to wait for a specific event instead of using a timeout ?

Reply:
@QuCMGisaia is it ok for you ?

Reply:
Yes that is perfect !